utils.js

```diff
@@ -0,0 +1,34 @@
+var fs = require('fs');
+var path = require('path');
+
+var defaults = require("lodash.defaults");
+
+// Utility functions
```

Comment:
The comment here isn't helpful. It's pretty clear that utils declares utility functions. However you can document each individual function if you want.